packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-graphql-schema.ts

```diff
@@ -460,8 +461,8 @@ export const getDirectiveDefinitions = async (context: $TSContext, resourceDir:
 const s3ResourceAlreadyExists = (): string | undefined => {
   try {
     let resourceName: string;
-    const amplifyMeta = stateManager.getMeta();
-    if (amplifyMeta[AmplifyCategories.STORAGE]) {
+    const amplifyMeta: $TSMeta = stateManager.getMeta(undefined, { throwIfNotExist: false });
```

Comment:
nit: is there a reason we avoid getProjectDetails?

Reply:
Yes, it loads the team-provider-info file even though we are not using it.